src/io/flutter/view/FlutterView.java

```diff
@@ -594,6 +594,42 @@ private void activateToolWindow() {
   }
 }
 
+class OpenDevToolsAction extends FlutterViewAction {
```

Comment:
Can this be unified with io.flutter.run.OpenDevToolsAction? If not, I'd prefer a different name for one of these action classes. E.g. OpenDevToolsMenuAction here or OpenDevToolsToolbarAction there.

Reply:
There is a bunch of overlap. It's slightly easier to have code dupe for these two classes, rather than trying to unify them; I renamed the one specific to the Flutter view to `FlutterViewDevToolsAction`.